test/integration/sds_dynamic_integration_test.cc

```diff
@@ -46,24 +46,27 @@ namespace Ssl {
 // Hack to force linking of the service: https://github.com/google/protobuf/issues/4221.
 const envoy::service::secret::v3::SdsDummy _sds_dummy;
 
-std::string sdsTestParamsToString(
-    const ::testing::TestParamInfo<std::tuple<Network::Address::IpVersion, Grpc::ClientType, bool>>&
-        p) {
-  return fmt::format(
-      "{}_{}_{}", std::get<0>(p.param) == Network::Address::IpVersion::v4 ? "IPv4" : "IPv6",
-      std::get<1>(p.param) == Grpc::ClientType::GoogleGrpc ? "GoogleGrpc" : "EnvoyGrpc",
-      std::get<2>(p.param) ? "UsesQuic" : "UsesTcp");
+struct TestParams {
+  Network::Address::IpVersion ip_version;
+  Grpc::ClientType grpc_type;
```

Comment:
`sds_grpc_type`? to make it clear that this relates to the SDS client, not the downstream client of the integration test

Reply:
done